### PR TITLE
Machine ID and GitHub Actions docs copy improvements

### PR DESCRIPTION
**Pull request by strideynet**
Many thanks to @alexfornuto for suggesting improvements here.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/machine-id/guides/github-actions-kubernetes.mdx [hunk 1]
```diff
@@ -27,12 +27,13 @@ Actions runners as well as GitHub Enterprise Server.
...
 
+(!docs/pages/includes/tctl.mdx!)
+
 - A Kubernetes cluster connected to your Teleport cluster. If you do not already
 have one configured, try our
 [Kubernetes Access getting started guide.](../../kubernetes-access/getting-started.mdx)
 In our examples, this Kubernetes cluster will be named `my-kubernetes-cluster`.
-- A local workstation with `tsh` and `tctl` authorized against your Teleport
-cluster as a user with the privileges to create configuration resources.
+- Your user should have the privileges to create token resources.
```
Comment (alexfornuto) [inline docs/pages/machine-id/guides/github-actions-kubernetes.mdx:36]: Should we clarify here if we're talking about Kubernetes or Teleport, or would it be obvious to the reader?

File: docs/pages/machine-id/guides/github-actions-kubernetes.mdx [hunk 2]
```diff
@@ -67,13 +68,17 @@ spec:
...
+      # This group will be attached to requests made by users with this role
+      # to the Kubernetes API. Configure a cluster role binding that binds this 
+      # group to a Kubernetes cluster role to grant privileges to this Teleport 
+      # role in the Kubernetes cluster.
       - github-demo
   deny: {}
 ```
 
+Note the value provided in `kubernetes_groups`. In a later step, you will bind
+this group to a Kubernetes role with a cluster role binding.
```
Comment (alexfornuto) [inline docs/pages/machine-id/guides/github-actions-kubernetes.mdx:80]: 👍

File: docs/pages/machine-id/guides/github-actions-kubernetes.mdx [hunk 3]
```diff
@@ -123,7 +129,7 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
-Apply this to your cluster using `kubectl`:
+Create this Cluster Role Binding in your Kubernetes cluster using `kubectl`:
```
Comment (alexfornuto) [inline docs/pages/machine-id/guides/github-actions-kubernetes.mdx:132]: Should this be:

```suggestion
Create this cluster role binding in your Kubernetes cluster using `kubectl`:
```

to match the casing seen above?

**Review by alexfornuto — APPROVED** [review]
A couple questions, but otherwise LGTM.

**Pull request merged**